generic/tests/cfg/iozone_windows.cfg

```diff
@@ -3,8 +3,11 @@
     only Windows
     type = iozone_windows
     cdrom_cd1 = isos/windows/winutils.iso
-    iozone_cmd = "%s:\Iozone\iozone.exe -azR -r 64k -n 1G -g 4G -M -b iozone.xls -f c:\testfile"
+    disk_letter = "C"
```

Comment:
Better use following way to check which disk is the boot disk
diskpart 
list volumes